### PR TITLE
add pagination limit docs

### PR DESCRIPTION
**Pull request by joan-ing**
Closes #

## Summary

- Add docs for [this PR](https://github.com/xataio/xata/pull/3117). 

---

### Remaining work

The following work still needs to be completed:

- [ ] <List outstanding tasks here>

### Dependencies

- [ ] <Tasks that must be completed before merging this pull request>

---

### Timing

**Release**:

- [x] When ready

### CODE REVIEW

File: 040-SDK/101-get.mdx [hunk 1]
```diff
@@ -209,7 +209,7 @@ const differentSizedPage = anotherPageWithDifferentSize.records;
 ```
 
 - `getAll()`: returns all the records in the query results. Warning: this is dangerous on large tables (more than 10,000 records), as it will potentially load a lot of data into memory and create a lot of requests to the server.
-- `getMany()`: returns the requested number of records for the query in a single response. The default is 20 records, but you can change it by modifying the pagination size.
+- `getMany()`: returns the requested number of records for the query in a single response. The default is 20 records, but you can change it by modifying the pagination size. The pagination limit is 50,000 records.
```
Comment (tsg) [inline 040-SDK/101-get.mdx:212]: I'm not sure if this is actually limited, `getMany()` might do cursor pagination behind the scenes. @SferaDev ?

File: 060-Rest-API/040-limits.mdx [hunk 2]
```diff
@@ -63,11 +63,11 @@ Xata applies some limits based on the plan level and the API endpoint in order t
...
 ## Response limits
 
-| Scope                   | Limit        |
-| :---------------------- | :----------- |
-| Page size               | 200 records  |
-| Offset pagination size  | 800 records  |
-| Cursor pagination depth | Unrestricted |
+| Scope                   | Limit          |
+| :---------------------- | :------------- |
+| Page size               | 200 records    |
```
Comment (tsg) [inline 060-Rest-API/040-limits.mdx:68]:
```suggestion
| Page size               | 1000 records    |
```

File: 040-SDK/101-get.mdx [hunk 3]
```diff
@@ -799,7 +799,7 @@ You can continue like this until `"more"` is returned as `false`.
 
 ### Offset-based Pagination
 
-The offset-page pagination is limited to querying up to 1000 records and is recommended only for usecases where you don't expect the number of records to grow beyond that. In most cases, you should use the cursor-based pagination.
+The offset-page pagination limit is 50,000 records (about 250 pages). In most cases, you should use the cursor-based pagination.
```
Comment (tsg) [inline 040-SDK/101-get.mdx:802]: I think we don't need the 250 pages limit, that depends on the page size, but what we actually limit is the page offset, which is limited to 49K.

```suggestion
The offset-page pagination limit is 50,000 records. In most cases, you should use the cursor-based pagination.
```

File: 040-SDK/101-get.mdx (hunk 1 above)
Comment (SferaDev) [reply 040-SDK/101-get.mdx:212]: Yeah here doesn't apply

Comment (SferaDev) [inline 040-SDK/101-get.mdx:212]:
```suggestion
- `getMany()`: returns the requested number of records for the query in a single response. The default is 20 records, but you can change it by modifying the pagination size.
```

**Review by SferaDev — APPROVED** [review]

**Pull request merged**